DotNet Examples/Test Monitor/DeleteResults/DeleteResults.cs

```diff
@@ -0,0 +1,60 @@
+﻿using NationalInstruments.SystemLink.Clients.TestMonitor;
```

Comment:
As we have two examples for Results now - can we make the folder structure as follow - 

`````
Test Monitor
--> Results 
------> Create Results
------> Delete Results

`````

Reply:
Renamed the `Results` folder to `CreateResultsAndSteps`